feat: inbox react hooks api reference (#696)
novuhq/docs: inbox/react/hooks/novu-provider.mdx

---
title: 'NovuProvider'
---

import NovuProviderExample from '/snippets/inbox/react/novu-provider-example.mdx';
import NovuParams from '/snippets/inbox/headless/api-reference/novu-params.mdx';

The `NovuProvider` is the top-level component that provides the [Novu instance](/inbox/headless/api-reference#novu) to the rest of the hooks through the context.

> Usually it's placed somewhere at the root of your application, which allows the hooks to be accessible throughout the application.

Reviewer: ```suggestion
Usually, it's placed somewhere in the root of your application, which makes the hooks accessible throughout the application.
```
